lib/java/com/google/android/material/button/MaterialButtonToggleGroup.java

```diff
@@ -661,9 +661,8 @@ private CornerData getNewCornerData(
       int index,
       int firstVisibleChildIndex,
       int lastVisibleChildIndex) {
-    int childCount = getChildCount();
     CornerData cornerData = originalCornerData.get(index);
-    if (childCount == 1 || firstVisibleChildIndex == lastVisibleChildIndex) {
```

Comment:
maybe add a comment that this condition means "only one visible child"

Reply:
I've added a comment, hope it makes sense